scripts/make_snapshot_nopandas.py

```
@@ -0,0 +1,122 @@
+import sys, os, csv, math
+import matplotlib.pyplot as plt
+
+csv_path = sys.argv[1] if len(sys.argv) > 1 else "bench_context/context_sweep_v2.csv"
+out_dir  = sys.argv[2] if len(sys.argv) > 2 else "snapshots"
+os.makedirs(out_dir, exist_ok=True)
+
+def to_float(x):
+    try: return float(x)
```

Ruff (0.15.0)

[error] Multiple statements on one line (colon)

(E701)